infra/notifications/src/mobilePush.js

```diff
@@ -555,10 +455,86 @@ class MobilePush {
     await this._sendInChunks(data, deviceType, mobileRegisters)
   }
 
+  async _sendToCountryCode(data, deviceType, countryCode) {
+    const [addresses] = await db.sequelize.query(
+      `
+    select distinct iden.eth_address as address
+      from identity iden, growth_event gevent 
+      where gevent.eth_address=iden.eth_address 
+        and gevent.type='MobileAccountCreated' 
+        and iden.country=:countryCode
+    `,
+      {
+        replacements: {
+          countryCode
+        }
+      }
+    )
+
+    if (!addresses.length) {
+      console.log('There are no addresses from', countryCode)
+      return
+    }
+
+    const mobileRegisters = await MobileRegistry.findAll({
+      where: {
+        deviceToken: { [Sequelize.Op.ne]: null },
+        deviceType,
+        ethAddress: {
+          [Sequelize.Op.in]: addresses.map(a => a.address.toLowerCase())
+        },
+        deleted: false,
+        'permissions.alert': true
+      },
+      order: [['updatedAt', 'DESC']] // Most recently updated records first.
+    })
+
+    await this._sendInChunks(data, deviceType, mobileRegisters)
+  }
+
+  async _sendToLangCode(data, deviceType, languageCode) {
+    const [addresses] = await db.sequelize.query(
+      `
+    select distinct p.eth_address as address
+      from growth_participant p, growth_event e 
+      where p.eth_address=e.eth_address 
+        and e.type='MobileAccountCreated' 
+        and p.data->>'language'=:languageCode
+    `,
+      {
+        replacements: {
+          languageCode
+        }
+      }
+    )
+
+    if (!addresses.length) {
+      console.log('There are no addresses from', languageCode)
+      return
+    }
+
+    const mobileRegisters = await MobileRegistry.findAll({
+      where: {
+        deviceToken: { [Sequelize.Op.ne]: null },
+        deviceType,
+        ethAddress: {
+          [Sequelize.Op.in]: addresses.map(a => a.address.toLowerCase())
+        },
+        deleted: false,
+        'permissions.alert': true
+      },
+      order: [['updatedAt', 'DESC']] // Most recently updated records first.
+    })
+
+    await this._sendInChunks(data, deviceType, mobileRegisters)
+  }
+
   async multicastMessage(data) {
```

Comment:
nit: in order to know what's going on on the server side, I would add a bit more logging throughout the code. at least something to track the progress of the notifications being sent. at least something like "Sending chunk X out of Y"